torch/csrc/jit/runtime/static/impl.cpp

```diff
@@ -46,14 +42,27 @@ StaticRuntime::StaticRuntime(const torch::jit::Module& m)
   }
 
   // remove unused input 0 from graph
-  if (graph_->inputs().at(0)->type()->is_module()) {
-    if (!graph_->inputs().at(0)->hasUses()) {
-      graph_->eraseInput(0);
+  if (g->inputs().at(0)->type()->is_module()) {
+    if (!g->inputs().at(0)->hasUses()) {
+      g->eraseInput(0);
```

Comment:
Here Input(0) is only removed in the case that it does not have any use. Do you know cases that Input(0) does have uses later in the graph?